### ISSUE TITLE
Fmd Updater doesn't work

### ISSUE THREAD

**Issue by Beylers** [riderkick/FMD#1017]
Version: 0.9.146.0 Windows 64 bit 

When the update notification pops up I press update, then it downloads it and the "Download finished " notification pops and I agree to restarting the process, but it doesn't do anything and fmd stays open. I close fmd and reopen it but it asks for the update again and it still doesn't restart after asking for it.

[fmd.log](https://github.com/riderkick/FMD/files/1735107/fmd.log)

**Comment by wuhugm** [comment]
The updater is fixed!

Thanks for adding the "enter to retry" in the cmd updater, @riderkick!

**Comment by Owyn** [comment]
same for me

**Comment by riderkick** [comment]
Do the updater show failed message. It does over you to "Press enter to retry", right?

**Comment by Beylers** [comment]
No, after it asks to restart because the download is finished it just closes the popup and fmd stays open as if nothing happened. Also uploaded a gif of it
![some](https://user-images.githubusercontent.com/13386163/36381262-5adfc8d2-1585-11e8-981f-838741b60de8.gif)

**Comment by riderkick** [comment]
Probably you don't have the updater.exe or fmd doesn't have permission to change the file. Before calling updater it will rename the updater to old_updater and call this file. But if there is no updater, its just exit without throw something. I should've put the error message here. You can try to update manually.

**Comment by Manabi** [comment]
This problem is still around. I manually updated a day or two ago to 0.9.149.0, and today it did this exact same thing when trying to update to 0.9.150.0.  Permissions are fine (it's in a PortableApps directory, not in Program Files) and updater.exe exists.  In fact, when I manually updated before today there was still an old_updater.exe file in the directory, which the updated FMD removed.  In between program updates, it updated website modules several times successfully.

I had to manually update the first time due to the _"7za.exe not found"_ error, even though 7za.exe was in FMD's directory and permissions were correct.  It's possible this is the underlying problem.
